client/src/components/dashboard/AddInventory.js

```diff
@@ -0,0 +1,116 @@
+import React, {useState} from 'react';
+import Axios from 'axios';
+import Modal from 'react-modal';
+
+
+const AddInventory = () => {
+    /******/
+  const [inventoryName, setInventoryName] = useState('');
+  const [inventoryAccess, setInventoryAccess] = useState('');
+  const [inventoryDate, setInventoryDate] = useState('');
+  const [inventoryCount, setInventoryCount] = useState('');
+  const [inventoryItems, setInventoryItems] = useState('');
+  const [inventoryOwner, setInventoryOwner] = useState('');
+  const [inventoryHistory, setInventoryHistory] = useState('');
+  const [inventoryDescription, setInventoryDescription] = useState('');
+
+  /******/
+  const [isSuccessModalOpen, setIsSuccessModalOpen] = useState(false);
+  const [successModalMsg, setSuccessModalMsg] = useState('');
+
+  const onSubmitInventory = (e) => {
+    e.preventDefault();
+    Axios.post('http://localhost:5000/api/inventory', {
+      name: inventoryName,
+      description: inventoryDescription,
+      owner: inventoryOwner,
+      access: inventoryAccess,
+      date: inventoryDate,
+      items: inventoryItems,
+      inventoryCount,
+      history: inventoryHistory
+    }).then(res => {
+      if (res.data.status == '1') {
+        setSuccessModalMsg(res.data.message);
+        setIsSuccessModalOpen(true)
+      }
+    });
+
+    setInventoryName('');
+    setInventoryDescription('');
+    setInventoryOwner('');
+    setInventoryAccess('');
+    setInventoryHistory('');
+    setInventoryDate('');
+    setInventoryCount('');
+    setInventoryItems('');
+
+  }
+
+
+  return (
+      <div>
+          <form onSubmit={(e) => onSubmitInventory(e)}>
+            <div className="field">
+                <label>Name</label>
+                <div className="control">
+                <input type="text" required value={inventoryName} onChange={(e) => setInventoryName(e.target.value)} />
```

Comment:
`setInventoryName()`
these could perhaps be changed to one function that accepts the source and target value... like 
`setInventoryItem(name, e.target.value)`